src/server/protocol.ts

```diff
@@ -681,9 +681,17 @@ namespace ts.server.protocol {
 
     export type OrganizeImportsScope = GetCombinedCodeFixScope;
 
+    export const enum OrganizeImportsMode {
+        All = "All",
+        SortAndCombine = "SortAndCombine",

```

Comment:
While the PR focuses on the `RemoveUnused`... I'm more excited about `SortAndCombine` :P I see that `skipDestructiveCodeActions` was a thing (didn't know about it), gonna give it a try in my projects for now

Reply:
Yeah, to be clear, `SortAndCombine` ought to be identical to setting `skipDestructiveCodeActions` which is already set in a VS Code code action called `sortImports`. (It’s not very discoverable because I learned recently that these code actions lack autocomplete support in the VS Code settings.json files.)

Reply:
Ye, I got the fact that this one isn't a new option - but I had no idea that it was already a thing. So I'm glad that I've discovered that old option through this PR :P

Reply:
I can't figure out how to actually use this old option in VS Code. I've found this in the source code:
https://github.com/microsoft/vscode/blob/3a8b1fe03ebbcf57fb9c50b161db91229e2fe04a/extensions/typescript-language-features/src/languageFeatures/organizeImports.ts#L50

I've tried a lot of combinations with `_typescript.*`, `typescript.*`, `source.*` and `*.organizeImports.sortOnly: true` but none of those work for me.

Reply:
```json
  "editor.codeActionsOnSave": {
    "source.sortImports": true
  }
```

Reply:
I believe Organize Imports is a VS Code standard command, and so automatically appears in the command palette when registered. These variations are registered so can be triggered by ID in configurations like `codeActionsOnSave`, but I think we would need to add them to the `contributes` of the extension package.json to show up in the command palette. @mjbvz is that correct?